### ISSUE TITLE
project_ownership release 2.0.1 crashes in redcap8.11.2 with php7.2

### ISSUE THREAD

**Issue by marlycormar** [ctsit/project_ownership#34]
I am unable to reproduce the fatal error I received before. However, when trying to enable the module, I get the error below but the module is still enabled:

```
An error occurred while enabling the module: 				<div class="red" style="margin:20px 0px;max-width:700px;">
					<b>REDCap crashed due to an unexpected PHP fatal error!</b><br><br>
					<b>Error message:</b> Uncaught Error: Class 'ProjectOwnership\ExternalModule\Exception' not found in /var/www.backup/warrior_stage/warrior_stage-20190130T1409/modules/project_ownership_v1.2.1/ExternalModule.php:190
Stack trace:
#0 /var/www.backup/warrior_stage/warrior_stage-20190130T1409/modules/project_ownership_v1.2.1/ExternalModule.php(137): ProjectOwnership\ExternalModule\ExternalModule->checkModuleDependencies()
#1 [internal function]: ProjectOwnership\ExternalModule\ExternalModule->redcap_module_system_enable('v1.2.1')
#2 /var/www.backup/warrior_stage/warrior_stage-20190130T1409/redcap_v8.11.2/ExternalModules/classes/ExternalModules.php(1351): call_user_func_array(Array, Array)
#3 /var/www.backup/warrior_stage/warrior_stage-20190130T1409/redcap_v8.11.2/ExternalModules/classes/ExternalModules.php(1439): ExternalModules\ExternalModules::startHook('project_ownersh...', 'v1.2.1', Array)
#4 /var/www.backup/warrior_stage/warrior_stage-20190130T1409/redcap_v8.11.2/ExternalModules/classes/ExternalModules.php(562): ExternalModules\Ex<br>
					<b>File:</b> /var/www.backup/warrior_stage/warrior_stage-20190130T1409/modules/project_ownership_v1.2.1/ExternalModule.php<br>
					<b>Line:</b> 190<br>
				</div>
				The 'hook_module_system_enable' hook did not complete for the 'project_ownership' module because of the following error:

Error Message: Uncaught Error: Class 'ProjectOwnership\ExternalModule\Exception' not found in /var/www.backup/warrior_stage/warrior_stage-20190130T1409/modules/project_ownership_v1.2.1/ExternalModule.php:190
Stack trace:
#0 /var/www.backup/warrior_stage/warrior_stage-20190130T1409/modules/project_ownership_v1.2.1/ExternalModule.php(137): ProjectOwnership\ExternalModule\ExternalModule->checkModuleDependencies()
#1 [internal function]: ProjectOwnership\Externa
```

**Comment by marlycormar** [comment]
Thank you for working on this so diligently @tbembersimeao!!

**Comment by marlycormar** [comment]
Issue fixed by PR #35. Thanks @tbembersimeao
